src/Screens/signup/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from "react";
+import {View,Text} from 'react-native'
+
+import styles from './styles'
+
+import {LoginBottomCard,LoginHeader, LoginInputContainer,LoginButton} from "components/login";
```

Comment:
If they are reusable in both login and signup, then give them meaningful names, Components named Login are being used in the Signup screen that doesn't fill right